docs/competition/livestream/scenes.md

```diff
@@ -0,0 +1,50 @@
+# Scenes
+
+This page documents a few of the core scenes which are used for livestreams. Not all of them are applicable to all events, but most are.
+
+The various assets useful for creating these scenes can be found [here](https://drive.google.com/drive/folders/1pr1vKpnonxFoO8O5chsH5DF7Vsd0Tfih?usp=sharing).
+
+## Technical difficulties
+
+Things occasionally go wrong during events, so wrong it may be necessary to cut the whole feed. In this case, we show a "Technical difficulties" screen, comprised of a single image, and mute audio whilst we work on a solution.
+
+## Countdown
+
+We will usually "go live" prior to the scheduled start time, so viewers know we're preparing, and to build some atmosphere. Before the event starts, we put up a countdown screen, showing a live countdown to the event start time.
+
+We have our own branded [stream countdown](https://github.com/srobo/stream-countdown) to handle this, which uses OBS' browser source to embed into the stream.
+
+## Thanks for watching
+
+When ending a stream, we finish with few seconds of a "Thanks for watching". This screen mostly just shows some quick bits of useful information on future events (if any). The exact content for this screen differs for each event, but is generally just a single image source.
+
+## Back soon
+
+Depending on the length of a livestream, we may take a break in the middle. Cutting the stream at this time would lead to a drop off in users, so instead we show a "Back soon" screen. This may be a simple image, or may be combined with the [overlay](./matches.md#overlay) to show the actual return time.
+
+## Cameras
+
+If the livestream is going to have a few key presenters, it's helpful to have a scene which shows their faces on stream. Depending on how the presenters are communicating, this may be implemented in a few different ways.
+
+If they're communicating using Google Meet, or another video chat service, a screen capture of that can be cropped and embedded into OBS.
+
+If they're in the same room, a simple video capture is enough.
+
+In the case there's a hybrid, it depends on the nature. Often webcam devices can't be captured by OBS and used in a video call. Depending on the layout and number of people, passing the webcam to Google Meet and capturing your own feed through it may be the best option.
```

Comment:
```suggestion
In the case there's a hybrid, it depends on the nature. Often webcam devices can't be captured by OBS and used in a video call simultaneously. Depending on the layout and number of people, passing the webcam to Google Meet and capturing your own feed through it may be the best option.
```